docs/content/guides/dagster/ml-ops.mdx

```diff
@@ -0,0 +1,211 @@
+    # Train XGBoost model, which is a highly efficient and flexible model
+    xgb_r = xg.XGBRegressor(
+        objective="reg:squarederror", eval_metric=mean_absolute_error, n_estimators=20
+    )
+    xgb_r.fit(transformed_X_train, transformed_y_train)
+    return xgb_r
+
+
+@asset
+def comments_model_test_set_r_squared(transformed_test_data, xgboost_comments_model):
+    transformed_X_test, transformed_y_test = transformed_test_data
+    # Use the test set data to get a score of the XGBoost model
+    score = xgboost_comments_model.score(transformed_X_test, transformed_y_test)
+    return score
+```
+
+We include [freshness policies](/concepts/partitions-schedules-sensors/asset-sensors#freshness-policy-sensors) in the models here which will ensure the upstream data transformation assets are updated to comply with these policies.
+
+### Evaluating our results
+
+In our model assets, we evaluated each of the models on the test data and in this case, got the [score](https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.XGBRegressor.score) derived from comparing the predicted to actual results. Next, to predict the results, we'll create another asset that runs inference on the model more frequently than the model is re-trained.
+
+```python file=/guides/dagster/ml_ops/ml_ops.py startafter=inference_start endbefore=inference_end
+@asset(freshness_policy=FreshnessPolicy(maximum_lag_minutes=60))
+def latest_story_comment_predictions(xgboost, Tfidf_Vectorizer):
+    # Get the max ID number from hacker news
+
+    latest_item = requests.get(
+        f"https://hacker-news.firebaseio.com/v0/maxitem.json"
+    ).json()
+    # Get items based on story ids from the HackerNews items endpoint
+    results = []
+    scope = range(latest_item - 100, latest_item)
+    for item_id in scope:
+        item = requests.get(
+            f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        ).json()
+        results.append(item)
+
+    df = pd.DataFrame(results)
+    if len(df) > 0:
+        df = df[df.type == "story"]
+        df = df[~df.title.isna()]
+    inference_x = df.title
+    # Transform the new story titles using the existing vectorizer
+    inference_x = Tfidf_Vectorizer.transform(inference_x)
+    return xgboost.predict(inference_x)
+```
+
+Depending on what the objective of your ML model is, you can use this data to set alerts, save model performance history, and trigger retraining.
+
+---
+
+## Where to go from here
+
+- Dagster is flexible so there could be many 'right' ways to implement your machine learning pipeline and different avenues to explore
+- Dagster intergrates with [MLflow](/\_apidocs/libraries/dagster-mlflow) that can be used to keep track of your models
+- Dagster integrates with [Weights & Biases](/\_apidocs/libraries/dagster-wandb) and an [example](https://github.com/dagster-io/dagster/tree/master/examples/with_wandb) which demonstrates how to use W\&B's artifacts with Dagster.
```

Comment:
A backslash is showing up when this is rendered

Reply:
This is still an issue for me as well